marbot-alb.yml

```diff
@@ -0,0 +1,243 @@
+---
+# Copyright 2018-2020 widdix GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'marbot.io: Application Load Balancer (ALB) monitoring (https://github.com/marbot-io/monitoring-jump-start)'
+Metadata:
+  'AWS::CloudFormation::Interface':
+    ParameterGroups:
+    - Label:
+        default: 'marbot endpoint'
+      Parameters:
+      - EndpointId
+      - Stage
+    - Label:
+        default: 'ALB'
+      Parameters:
+      - LoadBalancerFullName
+      - TargetGroupFullName
+    - Label:
+        default: 'Thresholds'
+      Parameters:
+      - ALB5XXCountThreshold
+      - ALBRejectedConnectionCountThreshold
+      - Target5XXCountThreshold
+      - TargetConnectionErrorCountThreshold
+Parameters:
+  EndpointId:
+    Description: 'Your marbot endpoint ID (to get this value: select a channel where marbot belongs to and send a message like this: "@marbot show me my endpoint id").'
+    Type: String
+  Stage:
+    Description: 'marbot stage (never change this!).'
+    Type: String
+    Default: v1
+    AllowedValues: [v1, dev]
+  LoadBalancerFullName:
+    Description: 'The full name of the load balancer (last part of ARN, e.g., app/load-balancer-name/1234567890123456).'
+    Type: String
+  TargetGroupFullName:
+    Description: 'The full name of the target group (last part of ARN, e.g., targetgroup/target-group-name/1234567890123456).'
+    Type: String
+  ALB5XXCountThreshold:
```

Comment:
What about using a percentage instead of a number?

Reply:
Yes, could be added in the future as a 2nd parameter. So you can either work with absolut or relativ numbers.